### ISSUE TITLE
Failed build from AUR

### ISSUE THREAD

**Issue by moeryomenko** [tildearrow/kwin-lowlatency#49]
[build.log](https://github.com/tildearrow/kwin-lowlatency/files/3747073/build.log)

**Comment by tildearrow** [comment]
This seems unusual. Please report your current:

- Qt version
- Plasma version
- KDE Frameworks version

**Comment by tildearrow** [comment]
Can you try deleting the src/ directory in /home/\<user\>/.cache/yay/kwin-lowlatency/?

**Comment by tildearrow** [comment]
I think I found the issue: missing qt5-script. Can you try building again? I have updated the package.

**Comment by nazarianin** [comment]
I have the same error. 
QT - 5.13.1
plasma-desktop - 5.17.0
kde - 19.08.2
kwin-lowlatency  - 5.17.0
All from Arch stable repos.

**Comment by tildearrow** [comment]
@nazarianin Can you please do `sudo pacman -Q`? I do not know what could be the issue anymore...

**Comment by nazarianin** [comment]
[pacman_q.txt](https://github.com/tildearrow/kwin-lowlatency/files/3755701/pacman_q.txt)

**Comment by tildearrow** [comment]
@nazarianin Thanks

@nazarianin @moeryomenko What happens if you try building the package without a helper (and/or after cleaning the build cache)?

**Comment by moeryomenko** [comment]
@tildearrow after rebuild, have same error
```
/usr/include/qt/QtCore/qmetatype.h:59:2: error: #error qmetatype.h must be included before any header file that defines Bool
   59 | #error qmetatype.h must be included before any header file that defines Bool
      |  ^~~~~
...
```
KDE 5.63.0
Plasma 5.17.1
```
$ qmake -v
QMake version 3.1
Using Qt version 5.13.1 in /usr/lib
```

**Comment by undu** [comment]
Same issue here, clean-building with `makepg -s`: [kwin_build.zip](https://github.com/tildearrow/kwin-lowlatency/files/3764438/kwin_build.zip)

`pacman -Q`: [pacman_q.log](https://github.com/tildearrow/kwin-lowlatency/files/3764457/pacman_q.log)

**Comment by tildearrow** [comment]
The issue may have been found! See issue #51 for details.

**Comment by tildearrow** [comment]
Fixed in v5.17.1-2!
